sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -56,7 +56,7 @@
 public class CryptographyAsyncClient {
     static final String KEY_VAULT_SCOPE = "https://vault.azure.net/.default";
     static final String SECRETS_COLLECTION = "secrets";
-    JsonWebKey key;
+    AtomicReference<JsonWebKey> key;
```

Comment:
What motivated your introduction of the `AtomicReference` type?

Reply:
On second thought, redesigned to keep it non atomic.